### PR TITLE
water and bots

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Makes AI cyborgs limited to 1 print pr AI core
Makes hunter water basions no longer run out of water
Vastly improves churchs water fountains to be less likely to run dry
Fixes borgs being perma stunned/weakened/paralized

### CODE REVIEW

File: code/game/objects/structures/watercloset.dm [hunk 1]
```diff
@@ -515,6 +515,19 @@
 			to_chat(user, SPAN_NOTICE("You empty the [O] into the [src]."))
 
 
+/obj/structure/sink/fountain
+	name = "water fountain"
+	desc = "A small fountain of polished stone and has a small pump at the bottom that helps push fresh water."
```
Comment (elgeonmb) [inline code/game/objects/structures/watercloset.dm:520]: "and" should probably be "that" here as well

File: code/modules/mob/living/silicon/Cyborg_AI/remote_control.dm [hunk 2]
```diff
@@ -63,6 +63,12 @@
 		to_chat(user, "<span class='warning'>The control subsystems are tasked to capacity; they cannot support any more cyborgs.</span>")
 		return
 
+	if(user.amount_of_borgs_printed >= 1)
+		to_chat(user, "<span class='warning'>Maxium AI borgs per Unit Dispenced.</span>")
```
Comment (elgeonmb) [inline code/modules/mob/living/silicon/Cyborg_AI/remote_control.dm:67]: *dispensed

Comment (elgeonmb) [reply code/modules/mob/living/silicon/Cyborg_AI/remote_control.dm:67]: also @maximum

Comment (elgeonmb) [reply code/modules/mob/living/silicon/Cyborg_AI/remote_control.dm:67]: also *maximum

**Comment by LiteralMushroom** [comment]
Should be split into separate "fix water" and "borg stuff" PRs.

File: code/game/objects/structures/watercloset.dm [hunk 3]
```diff
@@ -515,6 +515,19 @@
...
+	icon_state = "BaptismFont_Water"
+	refill_rate = 2
+
+/obj/structure/sink/basion
+	name = "water basion"
+	desc = "A deep basion of polished stone and has prefilled with fresh water."
```
[the remark this answers is not in the document]
Comment (ThePainkiller) [reply code/game/objects/structures/watercloset.dm:526]:
```suggestion
	desc = "A deep basin of polished stone that has been pre-filled with fresh water."
```

File: code/game/objects/structures/watercloset.dm (hunk 1 above)
Comment (ThePainkiller) [reply code/game/objects/structures/watercloset.dm:520]:
```suggestion
	desc = "A small fountain of polished stone that has a small pump at the bottom which helps pump out fresh water."
```

**Comment by ThePainkiller** [comment]
Dear god please, I don't want to recall the last time I saw the manifest FLOODED by one AI's army of shells

File: code/modules/mob/living/silicon/Cyborg_AI/remote_control.dm (hunk 2 above)
Comment (ThePainkiller) [reply code/modules/mob/living/silicon/Cyborg_AI/remote_control.dm:67]:
```suggestion
		to_chat(user, "<span class='warning'>Maximum AI borgs per unit already dispensed.</span>")
```

**Pull request merged**